cloverage/src/cloverage/report/console.clj

```diff
@@ -0,0 +1,105 @@
+(ns cloverage.report.console
+  (:gen-class)
+  (:require
+   [clojure.string :as cs]
+   [cloverage.report :refer [file-stats total-stats]]))
+
+(def ansi
+  (let [esc \u001b
+        reset "[0m"
+        tbl {:red "[31m"
+             :yellow "[33m"
+             :green "[32m"
+             :bright "[1m"}]
+    (fn [code text]
+      (when text
+        (str esc (get tbl code reset) text esc reset)))))
+
+(defn strip-ansi [text]
+  (when text
+    (cs/replace text #"\u001b\[\d+m" "")))
```

Comment:
`clojure.string/replace` will throw a null bomb,

Its like it is, so calling something like

```clojure
(print-table
  [:field1 :field2]
  [{:field1 "hello" :field2 "world"}
   {:field1 "hello"}]
  ["Field1", "Field2"])
```

would print a table with an empty cell rather than blowing up.

I guess it could be stemmed in `count-printable` instead?